bci_tester/data.py

```diff
@@ -340,14 +350,26 @@ def create_BCI(
     bci_type=ImageType.APPLICATION,
 )
 
-CONTAINER_389DS = create_BCI(
+CONTAINER_389DS_2_0 = create_BCI(
     build_tag="suse/389-ds:2.0",
+    available_versions=["15.4"],
+    bci_type=ImageType.APPLICATION,
+    healthcheck_timeout=timedelta(seconds=240),
+    extra_environment_variables={"SUFFIX_NAME": "dc=example,dc=com"},
+    forwarded_ports=[PortForwarding(container_port=3389)],
+)
+
+CONTAINER_389DS_2_2 = create_BCI(
```

Comment:
you need to add a marker to pyproject.toml for this

Reply:
done